src/controllers/reports/reportController.ts

```
@@ -0,0 +1,264 @@
+  try {
+    const user = req.user;
+    if (!user) {
+      throw new AuthenticationError('User not authenticated');
+    }
+
+    logger.info(`got a request to create report for user: ${user.username}`);
+
+    if (typeof req.body?.parameters === 'string') {
+      try {
+        req.body.parameters = JSON.parse(req.body.parameters);
+      } catch (e) {
```

ESLint

[error] 'e' is defined but never used. Allowed unused caught errors must match /^_/u.

(@typescript-eslint/no-unused-vars)